text/0000-Project-Goals-2024h2.md

```diff
@@ -0,0 +1,522 @@
+| *Org decision*                                                                                                                        |                  |                    |
+| ↳ [Async WG reorganization](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                         | [Niko Matsakis][]    |                    |
+| *RFC decision*                                                                                                                        |                  |                    |
+| ↳ [Patterns of empty types](https://rust-lang.github.io/rust-project-goals/2024h2/Patterns-of-empty-types.html#ownership-and-team-asks)                                                       | [@Nadrieril][]       |                    |
+| ↳ ["Send bound" problem](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                            | [Niko Matsakis][]    | ![Complete][]      |
+| ↳ [Async closures](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                                  | [Michael Goulet][] |                    |
+| ↳ [Derive smart pointer](https://rust-lang.github.io/rust-project-goals/2024h2/rfl_stable.html#ownership-and-team-asks)                                                                       | [Alice Ryhl][]        |                    |
+| ↳ [Ergonomic ref-counting](https://rust-lang.github.io/rust-project-goals/2024h2/ergonomic-rc.html#ownership-and-team-asks)                                                                   | [Jonathan Kelley][]      |                    |
+| *Secondary RFC review*                                                                                                                |                  |                    |
+| ↳ [Trait for async iteration](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                       | [Eric Holk][]           |                    |
+| *Stabilization*                                                                                                                       |                  |                    |
+| ↳ ["Send bound" problem](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                            | [Niko Matsakis][]    |                    |
+| *Stabilization decision*                                                                                                              |                  |                    |
```

Comment:
This list feels a bit long to me in terms of things we would commit to reviewing (going with the "start small" axiom).

I think the obvious candidate for removal is the one we haven't RFC'd yet and isn't part of a flagship goal, patterns of empty types (sorry @Nadrieril, feel free to make the case for it here). Of course it would still be good to consider it while it's fresh, if we have bandwidth for it.

Reply:
I do see how it's the odd one out. Tbh what I'm most interested in is t-lang saying "that sounds like something we want". The prioritization and resource commitments aren't required, this can take however long it takes.

Reply:
Are you proposing @tmandry that we remove this as a stabilization candidate? (It seems like a stretch to me regardless.)

I can move this to "Discussion and moral support".

Reply:
I don't for example know whether to include design meeting or RFC decision, sounds like we are saying "no".

Reply:
Sorry for the lack of clarity – I meant specifically that we should remove it as a stabilization candidate that we are committing to resolving this year. I'm still fine with scheduling a design meeting for it.